src/executable.ts

```diff
@@ -3,9 +3,37 @@
 import * as path from "path";
 import * as fs from "fs";
 
+export function getCodingAssistantAssetsPath(): string {
+  // Allow complete local override
+  if (process.env.SOURCERY_CODING_ASSISTANT_ASSETS_PATH) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Update `package.json` scripts `vscode_local` to set this env var

```js
		"vscode_local": "SOURCERY_EXECUTABLE=../core/run-sourcery.sh SOURCERY_CODING_ASSISTANT_ASSETS_PATH=../core/coding-assistant-app/dist/assets yarn run vscode",

Reply:
You don't need it - if `SOURCERY_EXECUTABLE` is set then the code below resolves it to the right place. I can include both, but I thought this would only be necessary for specific manual testing situations, where having it in the package json isn't necessary.

Reply:
Oh, that is nice ❤️